src/Common/Core/Impl/Imaging/IImageService.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Common.Core.Imaging {
+    /// <summary>
+    /// Provides access to images and icons
+    /// </summary>
+    public interface IImageService {
+        /// <summary>
+        /// Provides image based on one of the predefined types
+        /// </summary>
+        /// <returns>Platform-specific image source</returns>
+        object GetImage(ImageType imageType);
+
+        /// <summary>
+        /// Returns image source given name of the image moniker
+        /// such as name from http://glyphlist.azurewebsites.net/knownmonikers
+        /// when running in Visual Studio.
+        /// </summary>
+        /// <returns>Platform-specific image source</returns>
+        object GetImage(string name);
```

Comment:
`GetImage(ImageType imageType)` returngs predefined icons. Like 'function', 'variable' etc such as from editor `IGlyphService`. So it is consistent and the same. `GetImage(string name)` can be used to return custom `foo.png` or VS moniker based image.